skrifa/src/scale/scaler.rs

```diff
@@ -74,8 +74,9 @@ impl<'a> ScalerBuilder<'a> {
         self
     }
 
-    /// Adds the sequence of variation settings. This will clear any variations
-    /// specified as normalized coordinates.
+    /// Adds the sequence of variation settings.
+    ///
+    /// This will clear any variations specified as normalized coordinates.
```

Comment:
I think the type signature of this method is confusing enough I would include a few examples, or otherwise describe what types it is possible to use?

Reply:
Fair enough. It is a bit unwieldy. In these cases, I try to provide the most general API possible, hence the `IntoIterator` and `From` but that does make it hard to decipher what concrete types to actually pass to the method. I'll add examples to the docs.